.gitignore

```diff
@@ -2,3 +2,4 @@
 *~
 *.aar
 *.apk
+.vscode
```

Comment:
Is this intentional? If this is common, I'm fine.

Reply:
This should stay (imo). If you use vscode and you have configurations for the workspace (e.g. for modules) you need this folder.